admiral/pkg/clusters/types.go

```diff
@@ -145,6 +146,47 @@ type ServiceHandler struct {
 	ClusterID      string
 }
 
+func (sh *ServiceHandler) Added(obj *k8sV1.Service) {
+	log.Infof(LogFormat, "Added", "service", obj.Name, sh.ClusterID, "received")
+	err := HandleEventForService(obj, sh.RemoteRegistry, sh.ClusterID)
+	if err != nil {
+		log.Infof(err.Error())
```

Comment:
Should we use Warnf or Errorf instead?  (same comment for other related methods)